src/core/linux/X11.zig

```diff
@@ -86,6 +86,11 @@ pub fn init(
 ) !X11 {
     // TODO(core): return errors.NotSupported if not supported
     const libx11 = try LibX11.load();
+
+    // Xlibx11.XInitThreads must be called first
+    //
+    // if not, 'Unknown sequence number while processing queue' errors occur.
+    _ = libx11.XInitThreads();
```

Comment:
I'd like us to remove this from the codebase ASAP, since we don't have any multi-threading yet this _should_ be doing nothing :) so let's get it out and debug any issues that arise from removing it